### PR TITLE
Mtv 215 add content tags to adoc files

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
https://issues.redhat.com/browse/MTV-215

Add content tags to adoc files

### CODE REVIEW

File: documentation/modules/error-messages.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+:_content-type: REFERENCE
+[id="error-messages_{context}"]
+= Error messages
+
+This section describes error messages and how to resolve them.
+
+.warm import retry limit reached
+
+The `warm import retry limit reached` error message is displayed during a warm migration if a VMware virtual machine (VM) has reached the maximum number (28) of changed block tracking (CBT) snapshots during the precopy stage. You must delete some of the CBT snapshots from the VM and restart the migration plan.
```
Comment (stoobie) [inline documentation/modules/error-messages.adoc:13]: @sbeskin-redhat 
```suggestion
Displayed during a warm migration if a VMware virtual machine (VM) has reached the maximum number (28) of changed block tracking (CBT) snapshots during the precopy stage. You must delete some of the CBT snapshots from the VM and restart the migration plan.
```
Maybe change "some" to indicate that you need to delete enough such that the total is less than or equal to the maximum.

File: documentation/modules/making-open-source-more-inclusive.adoc [hunk 2]
```diff
@@ -3,6 +3,7 @@
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
 [preface]
+:_content-type: SNIPPET
```
Comment (stoobie) [inline documentation/modules/making-open-source-more-inclusive.adoc:6]: @sbeskin-redhat This is a concept.

File: documentation/modules/mtv-resources-and-services.adoc [hunk 3]
```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
+:_content-type: REFERENCE
```
Comment (stoobie) [inline documentation/modules/mtv-resources-and-services.adoc:5]: @sbeskin-redhat This looks like a concept module to me. I'd read it to understand about customer resources and services. I don't know if I'd refer back to this as a reference.

File: documentation/modules/rn-2.0.adoc [hunk 4]
```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Release_notes/master.adoc
 
+:_content-type: REFERENCE
```
Comment (stoobie) [inline documentation/modules/rn-2.0.adoc:5]: @sbeskin-redhat  I'd call this a concept.

File: documentation/modules/rn-2.1.adoc [hunk 5]
```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Release_notes/master.adoc
 
+:_content-type: REFERENCE
```
Comment (stoobie) [inline documentation/modules/rn-2.1.adoc:5]: @sbeskin-redhat I'd call this a concept.

File: documentation/modules/rn-2.2.adoc [hunk 6]
```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Release_notes/master.adoc
 
+:_content-type: REFERENCE
```
Comment (stoobie) [inline documentation/modules/rn-2.2.adoc:5]: Concept.

File: documentation/modules/virt-migration-workflow.adoc [hunk 7]
```diff
@@ -2,7 +2,8 @@
 //
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
-[id="virt-migration-workflow_{context}"]
+:_content-type: REFERENCE
```
Comment (stoobie) [inline documentation/modules/virt-migration-workflow.adoc:5]: I'd call this a concept.

**Review by stoobie — CHANGES_REQUESTED** [review]
@sbeskin-redhat OK, I finished. Most of the files look fine. There a are a few topics labeled as REFERENCE that look like concept topics to me. A few other minor comments or questions.

File: documentation/modules/error-messages.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply documentation/modules/error-messages.adoc:13]: Will open a separate bug for editing the module's content

File: documentation/modules/rn-2.0.adoc (hunk 4 above)
Comment (sbeskin-redhat) [reply documentation/modules/rn-2.0.adoc:5]: Why is it a concept? It just lists the changes made in the new version.

File: documentation/modules/rn-2.1.adoc (hunk 5 above)
Comment (sbeskin-redhat) [reply documentation/modules/rn-2.1.adoc:5]: Why is it a concept? It just lists the changes made in the new version.

File: documentation/modules/error-messages.adoc (hunk 1 above)
Comment (stoobie) [reply documentation/modules/error-messages.adoc:13]: @sbeskin-redhat Sounds good.

**Pull request merged**